client/dashboard/profile-wizard/steps/start/style.scss

```diff
@@ -0,0 +1,58 @@
+/** @format */
+
+.woocommerce-profile-wizard__body {
+	.woocommerce-profile-wizard__tracking {
+		margin-bottom: $gap-large;
+
+		label {
+			font-size: 14px;
+		}
+
+		.components-base-control {
+			display: inline-block;
+		}
+
+		.components-form-toggle {
+			display: none;
+		}
+
+		.components-base-control__field {
+			margin-bottom: 0;
+		}
+
+		@include breakpoint( '<782px' ) {
+			.components-form-toggle {
+				display: inline-block;
+			}
+			.components-checkbox-control__input {
+				display: none;
+			}
+		}
+	}
+
+	.woocommerce-profile-wizard__benefit {
+		display: flex;
+
+		svg {
+			width: 24px;
+			min-width: 24px;
+			margin-right: $gap-large;
+		}
+
+		.woocommerce-profile-wizard__benefit-title {
+			margin-top: 0;
+			margin-bottom: $gap-smaller;
+		}
+
+		p {
+			padding-bottom: $gap;
+			margin-top: 0;
+			border-bottom: 1px solid $muriel-gray-50;
+			font-size: 14px;
```

Comment:
The `font-size` here does not seem to be working. I'm getting a `16px` font-size from https://github.com/woocommerce/woocommerce-admin/blob/master/client/dashboard/profile-wizard/style.scss#L41-L42 instead.

Reply:
Same- got it fixed up.